### PR TITLE
ROX-9750: Add known issue to Release Notes

### PR DESCRIPTION
**Pull request by kcarmichael08**
- Version(s): rhacs-docs and rhacs-docs-3.69.1
- [Issue](https://issues.redhat.com/browse/ROX-9750)
- [preview](https://deploy-preview-44883--osdocs.netlify.app/openshift-acs/latest/release_notes/369-release-notes#known-issues-369)

### CODE REVIEW

**Comment by kcarmichael08** [comment]
Approved by SME (ACS has no QE): https://srox.slack.com/archives/D03CFP0JS2X/p1650650414160119

File: release_notes/369-release-notes.adoc [hunk 1]
```diff
@@ -83,8 +83,12 @@ In {product-title} 3.69, when Collector detects that the host is using UEFI secu
 === Scanner memory limit increases
 Red Hat has increased the default Scanner memory limit from `3000` MiB to `4` GiB.
 
+[id="known-issues-369"]
+== Known issues
+* *ROX-9750*: The `FROM` instruction in the `DISALLOWED DOCKERFILE LINE` policy field is not recognized by {product-title-short}. For example, creating a policy that disallows `FROM:unwanted.example.com` in the Dockerfile will not generate a policy violation.
```
Comment (michaelryanpeter) [inline release_notes/369-release-notes.adoc:88]:
```suggestion
* *ROX-9750*: The `FROM` instruction in the `DISALLOWED DOCKERFILE LINE` policy field is not recognized by {product-title-short}. For example, creating a policy that disallows `FROM:unwanted.example.com` in the Dockerfile does not generate a policy violation. 
```
[ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense) recommends avoiding future tense.

Comment (kcarmichael08) [reply release_notes/369-release-notes.adoc]: Fixed - thank you!

**Pull request merged**